include/k4a/k4atypes.h

```diff
@@ -280,6 +280,9 @@ typedef enum
  * \remarks
  * Binned modes reduce the captured camera resolution by combining adjacent sensor pixels into a bin.
  *
+ * \remarks
+ * If modes are added, update the k4a_depth_mode_strings array.
```

Comment:
These remarks go in to the public API documentation. This probably doesn't belong there.